packages/server/src/index.ts

```diff
@@ -78,6 +78,9 @@ export async function createServer(opts: ServerOpts) {
 				requestLogger.debug("Handling request");
 
 				const url = new URL(`${opts.server}/${ctx.params["*"]}`);
+				// Preserve original query parameters
```

Comment:
I would look into whether elysia already exposes the search params?

Reply:
fixed!